test/test_functional_guardrail.py

```diff
@@ -123,13 +130,15 @@ def test_guardrail_functionality(self):  # pylint: disable=R0201
         file_name = os.path.join(ini_path, "guardrails", "guardrail_test.txt")
         with open(file_name, 'r') as input_file:
             lines_test = input_file.readlines()
+            lines_test.pop(2)
         os.remove(file_name)
         ini_path = os.path.abspath(
             os.path.join(os.path.dirname(__file__), os.pardir, "test_resource", "sample_project"))
         expec_file = os.path.join(ini_path, "test_guardrails.txt")
         with open(expec_file, 'r') as input_file:
             lines_expec = input_file.readlines()
-        self.assertEqual(lines_expec, lines_test)
+        self.assertEqual((str(lines_expec).replace("\n", "")).replace(r"\\", os.sep),
```

Comment:
May be worth putting a comment why the strips are happening?

Reply:
added comment